lib/requestIterator.js

```diff
@@ -28,7 +31,13 @@ RequestIterator.prototype.nextRequestBuffer = function () {
 
 RequestIterator.prototype.move = function () {
   // get the current buffer and proceed to next request
-  const ret = this.currentRequest.requestBuffer
+  let ret = this.currentRequest.requestBuffer
+  if (this.reqDefaults.idReplacement) {
+    const contentLengthRegex = /\[<contentLength>\]/
+    const reqStr = ret.toString().replace(/\[<id>\]/g, hyperid())
+    const bodyLength = reqStr.split(contentLengthRegex)[1].trim().length
+    ret = new Buffer(reqStr.replace(contentLengthRegex, `${bodyLength}`))
```

Comment:
let's try adding a pad to hyperid. It does not grow indefinitely, but up to https://github.com/mcollina/hyperid/blob/master/hyperid.js#L4. Maybe as an option when we create the hyperid instance.